### PR TITLE
bugfix: command sign -nest -t[s], must timestamped new (created) sign…

### PR DESCRIPTION
**Pull request by yjh-styx**
…ature, nor current (first in chain)!

### CODE REVIEW

**Comment by mtrojnar** [comment]
LGTM.
@olszomal Could you also take a look?

File: msi.c [hunk 1]
```diff
@@ -660,17 +660,9 @@ static PKCS7 *msi_pkcs7_prepare(FILE_FORMAT_CTX *ctx, BIO *hash, BIO *outdata)
...
-            PKCS7_free(cursig);
-            return NULL; /* FAILED */
-        }
-        PKCS7_free(p7);
-        return cursig;
-    }
-    return p7;
+    if (ctx->options->nest)
+        ctx->options->prevsig = cursig;
+        return p7;
```
Comment (olszomal) [inline msi.c:665]:
```suggestion
   return p7;
```

**Review by olszomal — CHANGES_REQUESTED** [review]

**Comment by olszomal** [comment]
It looks good to me. Fixed bug with nesting timestamped signatures.
Thank you for your contribution.

**Comment by mtrojnar** [comment]
@yjh-styx Will you fix the indentation?

**Pull request merged**